config/configuration_notwin_test.go

```diff
@@ -0,0 +1,30 @@
+// +build !windows
```

Comment:
👍🏽convention is to call this file `configuration_test.go` and on Windows call it `configuration_windows_test.go`